cmd/kube-scheduler/app/options/options.go

```diff
@@ -150,13 +150,33 @@ func newDefaultComponentConfig() (*kubeschedulerconfig.KubeSchedulerConfiguratio
 // Flags returns flags for a specific scheduler by section name
 func (o *Options) Flags() (nfs cliflag.NamedFlagSets) {
 	fs := nfs.FlagSet("misc")
-	fs.StringVar(&o.ConfigFile, "config", o.ConfigFile, `The path to the configuration file. The following flags can overwrite fields in this file:
+	fs.StringVar(&o.ConfigFile, "config", o.ConfigFile, `The path to the configuration file. The affected parameters are as follows.
+The following flags can overwrite fields in this file:
+  --algorithm-provider
+  --policy-config-file
+  --policy-configmap
+  --policy-configmap-namespace
+  --use-legacy-policy-config
+The following flags will be not effective when set --config , we must move into this file:
```

Comment:
TMI. Instead, we can put a note in each of the flags description

Reply:
Oh, I missed this line from my first comment. Agree this is not necessary